src/transforms/grok_parser.rs

```diff
@@ -48,7 +56,18 @@ impl Transform for GrokParser {
         if let Some(value) = value {
             if let Some(matches) = self.pattern.match_against(&value) {
                 for (name, value) in matches.iter() {
-                    event.insert_explicit(name.into(), value.into());
+                    let name: Atom = name.into();
```

Comment:
```suggestion
                    let name = name.into();
```

I believe it should be able to infer the type here.